docs/docs/themes/api-reference.md

```diff
@@ -56,39 +55,86 @@ module.exports = themeOptions => {
 }
 ```
 
-Then, in your theme's `gatsby-node.js` you can access them as the second argument to `createPages`:
+While using the usual object export (`module.exports = {}`) in your theme means that you can run the theme standalone as it's own site, When using a function in your theme to accept options you will need to run the theme as part of an example site. See how the [theme authoring starter](https://github.com/gatsbyjs/gatsby/tree/master/themes/gatsby-starter-theme-workspace) handles this using Yarn Workspaces.
+
+### Accessing Options elsewhere
+
+Note that because themes are also plugins you can also access the options in any of the lifecycle methods taht you're used to. For example, in your theme's `gatsby-node.js` you can access the options as the second argument to `createPages`:
```

Comment:
```suggestion
Note that because themes are plugins you can also access the options in any of the lifecycle methods that you're used to. For example, in your theme's `gatsby-node.js` you can access the options as the second argument to `createPages`:
```